scripts/liquidation_bot/index.ts

```diff
@@ -45,19 +52,34 @@ async function main() {
   ) as Liquidator;
 
   let lastBlockNumber: number;
-
+  let loopsUntilUpdateAssets = updateAssetsDelay / loopDelay;
```

Comment:
we probably shouldn't reuse `loopDelay`, given that `loopDelay` is used in a very different manner.

why not just set `loopUntilUpdateAssets` to some arbitrary value like 100, or 1,000? so every 100 loops we'll check for new assets